conbench/app/_plots.py

```diff
@@ -186,41 +187,67 @@ def _source(
     # the tooltip?
     commit_messages = [d["message"] for d in data]
 
-    dates = [dateutil.parser.isoparse(x["timestamp"]) for x in data]
+    # Note(JP): isoparse() returns a `datetime.datetime` object. And I think
+    # that the `timestamp` property corresponds to the utc-local invocation (or
+    # finish) time of the corresponding benchmark case run (the `timestamp`
+    # property on the `BenchmarkCreate` schema in the Conbench API). Are these
+    # tz-aware or tz-naive but in UTC?
+    datetimes = [dateutil.parser.isoparse(x["timestamp"]) for x in data]
+    date_strings = [d.strftime("%Y-%m-%d %H:%M %Z") for d in datetimes]
 
-    points, means = [], []
+    points, values_with_unit = [], []
```

Comment:
A variable name change attempt towards more clarity. Not perfection, but improvement :)
`points` also needs a better name, I think.